### PR TITLE
Fix errors  in READMEs and unify help functions

### PR DESCRIPTION
**Pull request by DFolchA**
Hello team,

This PR fixes some mistakes in the readmes such as wrong script names and unifies the help function format for all the scripts.

Regards,
Daniel Folch

### CODE REVIEW

File: aix/generate_wazuh_packages.sh [hunk 1]
```diff
@@ -32,11 +32,11 @@ show_help() {
   echo
   echo "Usage: $0 [OPTIONS]"
   echo
-  echo "    -e Install all the packages necessaries to build the RPM package"
-  echo "    -b <branch> Select Git branch. Example v3.5.0"
-  echo "    -s <rpm_directory> Directory to store the resulting RPM package. By default: /tmp/build"
-  echo "    -p <rpm_home> Installation path for the package. By default: /var"
-  echo "    -h Shows this help"
+  echo "    -b, --branch <branch>              Select Git branch or tag e.g. $BRANCH"
```
Comment (alberpilot) [inline aix/generate_wazuh_packages.sh]: Wrong indentation

File: aix/generate_wazuh_packages.sh [hunk 2]
```diff
@@ -32,11 +32,11 @@ show_help() {
...
+  echo "    -e, --environment                 Install all the packages necessaries to build the RPM package"
+  echo "    -s, --store  <rpm_directory>        Directory to store the resulting RPM package. By default: /tmp/build"
```
Comment (alberpilot) [inline aix/generate_wazuh_packages.sh]: Wrong indentation

File: aix/README.md [hunk 3]
```diff
@@ -14,19 +14,19 @@ To build an AIX package, you need to download this repository and use the `gener
...
         Usage: $0 [OPTIONS]"
 
-            -e Install all the packages necessaries to build the RPM package"
-            -b <branch> Select Git branch. Example v3.5.0"
-            -s <rpm_directory> Directory to store the resulting RPM package. By default: /tmp/build"
-            -p <rpm_home> Installation path for the package. By default: /var"
-            -h Shows this help"
+            -b, --branch <branch>              Select Git branch or tag e.g. $BRANCH
+            -e, --environment                  Install all the packages necessaries to build the RPM package
+            -s, --store  <rpm_directory>        Directory to store the resulting RPM package. By default: /tmp/build
```
Comment (alberpilot) [inline aix/README.md]: Wrong indentation

File: aix/README.md [hunk 4]
```diff
@@ -14,19 +14,19 @@ To build an AIX package, you need to download this repository and use the `gener
     $ cd aix
     ```
 
-2. Execute the `generate_rpm_package.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
+2. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
```
Comment (alberpilot) [inline aix/README.md]: Wrong script name

File: aix/README.md (hunk 3 above)
Comment (DFolchA) [reply aix/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/164434a1b72bd763d090f7356fec52718bf10d99

File: aix/generate_wazuh_packages.sh (hunk 2 above)
Comment (DFolchA) [reply aix/generate_wazuh_packages.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/164434a1b72bd763d090f7356fec52718bf10d99

File: aix/generate_wazuh_packages.sh (hunk 1 above)
Comment (DFolchA) [reply aix/generate_wazuh_packages.sh]: Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/164434a1b72bd763d090f7356fec52718bf10d99

File: wpk/README.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ To build  a WPK package, it is necessary to generate a X509 certificate and CA,
     ```shellsession
     # openssl x509 -req -days 365 -in wpkcert.csr -CA wpk_root.pem -CAkey wpk_root.key -out wpkcert.pem -CAcreateserial
     ```
-3. Execute the `generate_wazuh_app.sh` script to build the package. There are multiple parameters to select, Here you can see all the different parameters:
+3. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select, Here you can see all the different parameters:
```
[the remark this answers is not in the document]
Comment (DFolchA) [reply wpk/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/164434a1b72bd763d090f7356fec52718bf10d99

File: aix/README.md (hunk 4 above)
Comment (DFolchA) [reply aix/README.md]: Done in https://github.com/wazuh/wazuh-packages/pull/219/commits/136c1c0f1ca215fcb68503ddf7160fa47794df00

**Pull request merged**